test/util/server/server.go

```diff
@@ -386,7 +386,7 @@ func IsServerHealthy(url url.URL) (bool, error) {
 		},
 	})
 
-	url.Path = "/healthz"
+	url.Path = "/healthz/ready"
```

Comment:
@stevekuznetsov that is true, but I'm seeing issues that look to be race condition related because we are trying to test before the api server/controllers are actually ready. Testing against healthz/ready reduced the frequency of the occurrences, but I'm still trying to find a way to verify that the controllers have initialized before letting at least some of the tests go wild.

In particular, I'm seeing a failure with TestOadmPodNetwork and a large chunk of the Router tests.